halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/dockerRegistry/DockerRegistryAddAccountCommand.java

```diff
@@ -81,10 +86,19 @@ protected String getProviderName() {
   protected Account buildAccount(String accountName) {
     DockerRegistryAccount account = (DockerRegistryAccount) new DockerRegistryAccount().setName(accountName);
 
+    if (passwordFile != null) {
+      File pwdFile = new File(passwordFile);
+
+      if (pwdFile.exists() && !pwdFile.isDirectory() && pwdFile.canRead()) {
+        try {
+          password = new String(Files.readAllBytes(Paths.get(passwordFile)));
```

Comment:
Not a big fan of this, users generally tend to use passwordFile to avoid directly storing their password in the halconfig.